app/email.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/url"
 
 	l4g "github.com/alecthomas/log4go"
+	"github.com/cpanato/html2text"
```

Comment:
Why are we using a specialized version of the library? Can we avoid this? If not, it should be hosted in the Mattermost org.